documentation/faqs.md

```diff
@@ -6,62 +6,69 @@ title: FAQs
+## Can Reclaim Policy be set to `Retain` on dynamic PVs once volumes are provisioned with default reclaim policy `delete`?
+If the volume was dynamically provisioned, then the default reclaim policy is set to “delete”. This means by default, when the PVC is deleted, the underlying PV and storage asset will also be deleted.
+If volume needs to be retained, then reclaim policy needs to be changed from “delete” to “retain”. Please refer this [link](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/) for more details.
+
+
+## How to resize dynamic volumes once provisioned?
 Support for resizing existing dynamic volume is not yet there.
-Proposal is out for [review.](https://github.com/gnufied/community/blob/91b41028182a5291b4eccbf88f8065f66b2b7eed/contributors/design-proposals/grow-volume-size.md)
+Proposal is avaiallbe [here](https://github.com/gnufied/community/blob/91b41028182a5291b4eccbf88f8065f66b2b7eed/contributors/design-proposals/grow-volume-size.md). vSphere Cloud Provider does not support resize of volume. Please track this [issue](https://github.com/vmware/kubernetes/issues/168).
+
+
+## Is ReadWriteMany volume supported with vSphere Cloud Provider?
+ReadWriteMany volume is not supported by vSphere Cloud Provider.
+
 
-## Can we create ReadWriteMany volumes with VSphere storage, with pods on different machines?
-ReadWriteMany is not supported with Pods on different machine. This is supported on the collocated pods on the same node.
+## Is it mandatory to have all the node VMs in the same datastore?
+It is not mandatory to keep all node VMs on the same datastore. But make sure node VM has access to volumes datastores.
 
-## Is it mandatory to have all the machines in the same datastore? If so, it's a very strong limitation for us.
-It is not mandatory to keep all node VMs on the same datastore.  But make sure node VM has access to volumes datastores.
 
+## How can disk.EnableUUID enabled on Node VMs? 
+To enable disk.EnableUUID option for the Node VM, Select VM and select `Edit Settings`. Find
+Customize hardware -> VM Options -> Configuration Parameters -> Edit Configuration. Check to see if the parameter disk.EnableUUID is set, if it is there then make sure it is set to TRUE. If the parameter is not there, select Add Row and add it. This can be set on the template VM before cloning node VMs from the template.
 
-## Can we have the disk uuid enabled by default, so we don't need to do it machine by machine? What would be the risks of having it by default.
-You can enable disk UUID by default, while creating VM. Just add this parameter at
-Customize hardware -> VM Options -> Configuration Parameters -> Edit Configuration
 
 ## Is it mandatory to have all the machines in the same directory?
 Current code base requires Master and Node VMs to be present under one VM folder. Each Kubernetes Cluster deployed in vSphere, should be placed in their respective VM folder, else under root folder.
```

Comment:
Can you rename the question from Can Kubernetes Cluster span across multiple vCenters? to 
Can vSphere cloud provider Kubernetes Cluster spanning across multiple vCenters?

Also add question for
Can vSphere cloud provider Kubernetes Cluster spanning across multiple Datacenters?

Reply:
Fixed.